src/main/java/org/assertj/core/internal/Diff.java

```diff
@@ -66,11 +67,29 @@
     return diff(actual.toPath(), expected, charset);
   }
 
+  @VisibleForTesting
+  public List<Delta<String>> diff(Object[] actual, Object[] expected) throws IOException {
```

Comment:
Add unit tests for this method.

Reply:
let's get replace the `IOException` with `RuntimeIOException` as per my other comment.